Swiftfin/Views/AdminDashboardView/ServerUserDetailsView/ServerUserDetailsView.swift

```diff
@@ -43,13 +71,26 @@ struct ServerUserDetailsView: View {
                             router.route(to: \.resetUserPassword, userId)
                         }
                 }
-            }
-
-            Section(L10n.advanced) {
                 ChevronButton(L10n.permissions)
                     .onSelect {
                         router.route(to: \.userPermissions, viewModel)
                     }
+                ChevronAlertButton(
```

Comment:
Username should probably be at the top of this section.

Reply:
Do we want Username as a button or should I add it to the `.confirmationDialog` when selecting the profile picture? I used to have it there but I removed it. I'm not sure the button right below the Picture & Username VStack is good. For whatever reason UX/UI is always a struggle for me haha.

Reply:
The confirmation dialog is for whether to delete or set the image. Adding modification of the username there doesn't fit what it's being used for.